config/webpack.js

```diff
@@ -12,7 +12,8 @@ const NotifyPlugin = require('notify-webpack-plugin');
 const ASSET_NAME_PATTERN = 'static/[name]-[hash:6].[ext]';
 
 module.exports = (options = {}) => {
-  const iconFlavor = options.isProd ? 'prod' : 'beta';
+  // TODO: we need a way of specifying the actual prod config!
+  const iconFlavor = options.isProd ? 'beta' : 'dev';
```

Comment:
`env.blah` lets us set whatever options we want, so we could definitely change this to like `--env.env=dev|beta|prod`